build-system/server.js

```diff
@@ -33,6 +33,14 @@ var url = require('url');
 app.use(bodyParser.json());
 app.use(morgan('dev'));
 
+app.use('/', function(req, res, next) {
+  res.setHeader('AMP-Access-Control-Allow-Source-Origin',
```

Comment:
I could probably consolidate with code at line 298.
Also, instead of adding the header to all responses, shall we be more specific to your use case?